CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -223,15 +243,19 @@ class GroupsViewController: UIViewController {
     /// Updates the enabled state of next button based on the state of selectedGroups.
     private func updateNext() {
         nextButton.isEnabled = selectedGroups.count > 0
-        nextButton.backgroundColor = nextButton.isEnabled ? .backgroundRed : .backgroundLightGray
+        nextButton.backgroundColor = nextButton.isEnabled ? .backgroundOrange : .inactiveGreen
     }
 
     @objc func nextButtonPressed() {
-        print("Next button pressed.")
+        userDefaults.set(true, forKey: Constants.UserDefaults.onboardingCompletion)
+        let homeVC = HomeViewController()
+        navigationController?.pushViewController(homeVC, animated: true)
     }
 
-    @objc func backButtonPressed() {
-        delegate?.backPage(index: 1)
+    @objc func skipButtonPressed() {
+        userDefaults.set(true, forKey: Constants.UserDefaults.onboardingCompletion)
+        let homeVC = HomeViewController()
+        navigationController?.pushViewController(homeVC, animated: true)
```

Comment:
are these two functions doing the same thing?

Reply:
One allows users to skip the last page without entering the organizations that they are in. I assume we will be handling the two differently in the future. Right now the code is the same because nextButton will not be activated until a condition is met while the skipButton is always activated. But both should bring user to home page.